hrms/hr/doctype/interview/interview.js

```diff
@@ -221,6 +221,7 @@ frappe.ui.form.on("Interview", {
 				interview_round: frm.doc.interview_round || "",
 			},
 			callback: function (r) {
+                frm.clear_table("interview_details");
 				r.message.forEach((interviewer) =>
 					frm.add_child("interview_details", interviewer),
 				);
```

Comment:
```suggestion
				frm.clear_table("interview_details");
				r.message.forEach((interviewer) =>
					frm.add_child("interview_details", interviewer),
				);
```

Reply:
@SherinKR please commit this change. I don't have permission to push to your branch